src/application/redux/selectors/balance.selector.ts

```diff
@@ -9,5 +10,12 @@ export type BalancesByAsset = { [assetHash: string]: number };
  */
 export function balancesSelector(state: RootReducerState): BalancesByAsset {
   const utxos = Object.values(state.wallet.utxoMap);
-  return balances(utxos);
+  const balancesFromUtxos = balances(utxos);
+
+  const lbtcAssetHash = lbtcAssetByNetwork(state.app.network);
+  if (balancesFromUtxos[lbtcAssetHash] === undefined) {
```

Comment:
hasOwnProperty safer?

Reply:
Problem is that balancesFromUtxos is not an object (for Typescript) but a `Record`. However we could use the `in` operator I think but `===` always work as far as I know.

Reply:
You right but Typescript linter does not allow to use builtin method on "TS type" like `Record`:
```
Do not access Object.prototype method 'hasOwnProperty' from target object  no-prototype-builtins
```

Reply:
https://stackoverflow.com/questions/39282873/how-do-i-access-the-object-prototype-method-in-the-following-logic/39283005#39283005

Reply:
not nice syntax, but should be safer

Reply:
@Louisinger

Reply:
```suggestion
  if (!Object.prototype.hasOwnProperty.call(balancesFromUtxos, lbtcAssetHash)) {
```